src/mscorlib/src/System/String.cs

```diff
@@ -1441,6 +1441,16 @@ internal String[] SplitInternal(char[] separator, int count, StringSplitOptions
 
             return s;
         }
+                
+        [System.Security.SecuritySafeCritical]  // auto-generated
+        unsafe internal int GetBytesFromEncoding(byte* pbNativeBuffer, int cbNativeBuffer,Encoding encoding)
```

Comment:
There is another method called CreateStringFromEncoding whihc we are using now right above this method. GetBytesFromEncoding from encoding is opposite of CreateStringFromEncoding  and i think it's preferable to have both at the same place , unless someone says we don't want  a new internal method in string class.